02.calendar/cal.rb

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env ruby
+
+require 'optparse'
+
+input = ARGV.getopts('y:', 'm:')
+
+now = Time.now
+
+# use now if input not available
+year = input['y'] ? input['y'].to_i : now.year
+month = input['m'] ? input['m'].to_i : now.month
+
+this_month = false
+if now.year == year && now.month == month
+  this_month = true
+end
+
+# Get first day of the month
+start_day = Time.new(year, month).wday
+
+# Get last date of the month
+dates_in_month = (Time.new(year, (month % 12) + 1) - 1).day
+
+# Print input calendar date
+puts '      ' + month.to_s + '月　' + year.to_s
+
+# Print day of week
+puts ' 日 月 火 水 木 金 土'
```

Comment:
```suggestion
puts '日 月 火 水 木 金 土'
```

calコマンドに倣うと、一番左のスペースは不要ですね。
それによって、日付を表示する時も全体的に左に詰める必要がありますね。